lib/environment_light.rb

```diff
@@ -11,5 +11,13 @@
 require 'bundler/setup'
 
 # Load application settings
-require File.expand_path("../../config/initializers/rails_config", __FILE__)
-Settings = RailsConfig.load_files ["settings.yml","settings.local.yml", "settings/#{Rails.env}.yml"].map{|f| "#{Rails.root}/config/#{f}" }
+require Rails.root.join('config/initializers/rails_config.rb')
+Settings = RailsConfig.load_files([
+    'settings.yml',
+    "settings/#{Rails.env}.yml",
+    "environments/#{Rails.env}.yml",
+    'settings.local.yml',
+    "settings.#{Rails.env}.local.yml",
+    "environments/#{Rails.env}.local.yml",
+    'hets.yml'
+  ].map { |f| Rails.root.join('config', f) })
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.